packages/api-db-mongodb/src/db/schema.ts

```diff
@@ -239,3 +242,20 @@ export interface DBPageHistoryRevision extends DBPageRevision {
   _id: any
   articleID: string
 }
+
+export interface DBMemberPlan {
+  _id: any
+
```

Comment:
Remove line-breaks

Reply:
they are there on purpose for better readability